examples/animation/animate_decay.html

```diff
@@ -40,8 +40,14 @@
     <link rel="prev" title="matplotlib.animation.Animation.to_html5_video" href="../../api/_as_gen/matplotlib.animation.Animation.to_html5_video.html" /> 
 
 
-  </head>
+    </head>
   <body>
+    <div id="unreleased-message">
+      <link rel="next" title="artist Module" href="../artist_api.html" />
```

Comment:
And why the extra `link` (shouldn't it be in `head`?)

Reply:
Thanks, both should be fixed now - sorry bad cut and paste.